src/cli/index.js

```diff
@@ -39,16 +39,16 @@ process.on("unhandledRejection", (reason, p) => {
     }
 });
 
-exports.run = () => {
-    const program = new Command();
+exports.run = (opts = {}) => {
+    const program = new Command(opts.cliName || "testplane");
```

Comment:
Why do we need this?

Reply:
And still, why do we need this? What for?

Reply:
[src/cli/index.js](https://github.com/gemini-testing/hermione/pull/892/commits/d062638402a43920027c0b0c89eeb81fe9584167#diff-f00e7c78fa83a1316c2844a481cc2eeda7c7c2c4f958d39e9e457ddbc6b5c20f) used twice:

In bin/hermione and bin/testplane.

In future we will delete ./bin/hermione and opts in src/cli/index.js::run() also.

Right now, I introduced opts to pass cliName to compose proper help in CLI.

I decided to use opts as object with cliName field to make in more readable in on the call side.

Compate variants:

```
.run('hermione');
.run({ cliName: 'hermione' });
```

…for me second is much better.

Reply:
I mean, why do we need to pass `"hermione" / "testplane"` into Command constructor?